tests/test_intercept.js

```diff
@@ -510,7 +499,7 @@ test('isDone', async t => {
   scope.done()
 })
 
-test('request headers exposed', async t => {
+test('request headers exposed', t => {
```

Comment:
This looks like an error I made before; this test shouldn't be `async`. This is important because `tap` will auto-end `async` tests; it's important the test is done when control reaches the end and the promise resolves.